defaults/main.yml

```
@@ -12,7 +12,7 @@ galaxy_manage_paths: no
 galaxy_manage_clone: yes
 galaxy_manage_download: no
 galaxy_manage_static_setup: yes
-galaxy_manage_mutable_setup: yes
+galaxy_manage_managed_setup: yes
```

```suggestion
galaxy_manage_managed_setup: "{{ galaxy_manage_mutable_setup | default(yes) }}"
```

maybe something like this to ensure that people who don't read the upgrade notice can use the legacy value? @natefoo what are your opinions on that. "Read the release notes or else?"

---

I have no objections. @natefoo - up to you?

---

Yes, and we'll need to default all the other changed vars as well.

---

Coolie. Sorry for the extra work, but figure it'll be a bit more userfriendly.

---

It shouldn't be too bad as you can do:

```yaml
galaxy_managed_data_dir: "{{ galaxy_mutable_data_dir }}"
```

Rather than defaulting each instance of its usage.